pandas/io/formats/style_render.py

```diff
@@ -1571,7 +1703,7 @@ def _element(
     """
     Template to return container with information for a <td></td> or <th></th> element.
     """
-    if "display_value" not in kwargs:
+    if "display_value" not in kwargs or kwargs["display_value"] is None:
```

Comment:
When would kwargs["display_value"] be None?

Reply:
if index's name is None or if `self.hide_column_names`. Basically i just follow the check of `value`.